src/Grpc.Net.Client/Internal/GrpcCallSerializationContext.cs

```diff
@@ -295,6 +295,15 @@ private ReadOnlySpan<byte> CompressMessage(ReadOnlySpan<byte> messageData)
             return output.GetBuffer().AsSpan(0, (int)output.Length);
         }
 
+        private sealed class NonDisposableMemoryStream : MemoryStream
```

Comment:
This is just a "mini-type", but above on the server-side the same type is defined. Should this two definitions be merged to common place?